frameit/lib/frameit/editor.rb

```diff
@@ -484,12 +491,6 @@ def fetch_text(type)
 
       # No string files, fallback to Framefile config
       text = fetch_config[type.to_s]['text'] if fetch_config[type.to_s] && fetch_config[type.to_s]['text'] && fetch_config[type.to_s]['text'].length > 0 # Ignore empty string
-
-      if type == :title && !text
-        # title is mandatory
-        UI.user_error!("Could not get title for screenshot #{screenshot.path}. Please provide one in your Framefile.json or title.strings")
```

Comment:
Also: Is this the only context where `fetch_text` is used? Or should this "exit" maybe be kept for other uses?

Reply:
The only other place `fetch_text` is used is to get the value in the method `build_text_images`.

I think this is OK since as `fetch_text(:title)` will now return null if a title is not defined and then `build_text_images` would then just skip adding the title at: https://github.com/fastlane/fastlane/blob/22f845f82317933635044d55d13d146c8afae431/frameit/lib/frameit/editor.rb#L372.

And this scenario is also not currently possible to get into, since the framing will be skipped altogether via the new `should_skip?` check in `runner.rb`